tests/unit/v1/test_client.py

```diff
@@ -36,26 +37,42 @@ def _make_credentials():
 
 def _make_default_client(*args, **kwargs):
```

Comment:
it would be great to add one additional test without parameterization testing client initialization with:
1. not setting the database
2. setting the database
3. setting the database to None

and asserting the database is set correctly.

Reply:
@kolea2 there are 2 approaches to tackle this:

1. (current - neater approach but tests only client initialization) Adds `test_client_constructor_without_db` where DB isn't set and the database is asserted to equal `DEFAULT_DATABASE`

2. (Code become less readable, but tests are more exhaustive) we can add a check in `_make_default_client` to not use `database` param when it's an empty string during the client initialization. 

I choose the first approach for readability, but I have no qualms with the second one too. Leaving this comment open for now for Python reviewers.